packages/profile-sync-controller/CHANGELOG.md

```diff
@@ -11,12 +11,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Peer dependencies `@metamask/keyring-controller` and `@metamask/network-controller` are no longer also direct dependencies ([#5464](https://github.com/MetaMask/core/pull/5464)))
 
-## [10.1.0]
-
-### Added
-
-- Add primary SRP switching support for `AuthenticationController` and `UserStorageController` ([#5478](https://github.com/MetaMask/core/pull/5478))
-
```

Comment:
🤔 not sure I follow why this change is in here?

Reply:
D'oh 🤦🏻 This is not supposed to be here. I will run through the diff again.